lib/interfaces/include/MessageInterface.h

```diff
@@ -0,0 +1,55 @@
+#ifndef __MESSAGE_INTERFACE_H__
+#define __MESSAGE_INTERFACE_H__
+
+#include "FlexCAN_T4.h"
+#include <Arduino.h>
+#include "hytech.h"
+
+#ifndef CAN_Enabled
+// Un-comment line below if using CAN
+#define CAN_Enabled
+#endif
+
+#ifndef ETHERNET_Enabled
+// Un-comment line below if using Ethernet
+//#define ETHERNET_Enabled
+#endif
```

Comment:
is there a reason to have these? doesnt look like it is being used and we should start out using macros for this anyway.